### DISCUSSION TITLE
rConfig V6

### DISCUSSION THREAD

**Discussion by stephenstack** [patternfly/patternfly#5082] [Show and tell]
Please let me know your thoughts on the rConfig v6 implementation using Patternfly v4 100% for GUI.  I did use patternfly v3 for rConfig v5, but that has been deprecated since May this year. 

https://www.rconfig.com/ 
https://v6demo.rconfig.com/ (admin@domain.com/ admin)

**Comment by mcoker** [comment]
@stephenstack thanks for sharing, this looks great! Were you looking for any sort of feedback in particular? Also if you have any feedback - what worked well, what didn't, suggestions, or otherwise - we'd love to hear it!

**Comment by stephenstack** [comment]
@mcoker Nope, just sharing, and thank you all for your great work. I work with a lot of CSS UI frameworks over the years. Patternfly is an excellent enterprise professionally style framework. Only feedback from me, is the setup experience could be a little better for developers. I've imported via NPM to a Laravel application with Webpack/ Mix and VUE. It all works really well, but it was a learning curve to get it all setup (especially with the add-on classes).

**Comment by nicolethoen** [comment]
@stephenstack this is awesome! Would you ever be interested or willing to present your UI and talk a little bit about your experience and feedback at one of the PatternFly community meetings? It's always awesome to see the various web apps and UIs built with PatternFly at those meetings.

**Comment by stephenstack** [comment]
@nicolethoen would be delighted to present and discuss it. let's connect offline and arrange it.

**Reply by nicolethoen** [reply]
Are you in the patternfly slack workspace? I believe my email address is on my github profile as well. 
Not sure the easiest way to take this offline :)
